### PR TITLE
video details added and updated accordingly

### PR DESCRIPTION
**Pull request by daiyankhan**

### CODE REVIEW

File: scrap.js [hunk 1]
```diff
@@ -58,9 +60,18 @@ const getChannelVideos = async (channelId, maxPage)  => {
   const url = `https://m.youtube.com/channel/${channelId}/videos`
   const videos = parseVideos(await getURLContents(url, maxPage));
   console.log('video list', videos, `Total ${videos.length} videos`);
+
   return videos;
 }
 
+const getVideoDetails = (videos = []) => {
+  return videos.map(async (videoId) => {
```
Comment (thehungrycoder) [inline scrap.js]: let's construct it inside this function `const youtube = new Youtube();`

File: scrap.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 const puppeteer = require('puppeteer');
 const cheerio = require('cheerio');
 const iPhone = puppeteer.devices['iPhone X'];
+const { Youtube } = require('scrape-youtube');
+const youtube = new Youtube();
```
Comment (thehungrycoder) [inline scrap.js]: move this constructor to the function that's using it!

**Comment by thehungrycoder** [comment]
seems there is conflict in scrap.js. let's fix that. 

you don't need to create new pr for each change. just stay in same branch and push after fixing. 

let me know if you need any help!

File: scrap.js (hunk 2 above)
Comment (daiyankhan) [reply scrap.js]: done

File: scrap.js (hunk 1 above)
Comment (daiyankhan) [reply scrap.js]: not sure about this ! I have moved `const youtube = new Youtube();` inside of the function `getVideoDetails` as per comment above

File: scrap.js [hunk 3]
```diff
@@ -58,9 +60,20 @@ const getChannelVideos = async (channelId, maxPage)  => {
   const url = `https://m.youtube.com/channel/${channelId}/videos`
   const videos = parseVideos(await getURLContents(url, maxPage));
   console.log('video list', videos, `Total ${videos.length} videos`);
+
   return videos;
 }
 
+const getVideoDetails = (videos = []) => {
+  const youtube = new Youtube();
```
Comment (thehungrycoder) [inline scrap.js:93]: 👍

File: scrap.js (hunk 1 above)
Comment (thehungrycoder) [reply scrap.js]: yes, i meant that!

**Comment by thehungrycoder** [comment]
Great job. Thanks.

**Comment by daiyankhan** [comment]
THANK YOU ! will do better next time iA

**Pull request merged**